Amp 125382 marketing analytics docs (#557)
amplitude/amplitude-docs: content/collections/analytics/en/web-analytics.md

## Analyze industry-standard metrics
Out-of-the-box Marketing Analytics offers four sets of insights with default metrics, as well as optional goals or key outcomes, that you are driving users to:

- **Traffic by Channel**: Provides an overview of users visiting your product by channel. Metrics include visitors, bounce rate, session totals, average session duration, sessions per user, and marketing analytics views.
- **Traffic by Campaign**: Provides an overview of the users visiting your product by campaign. Metrics include visitors, bounce rate, session totals, average session duration, sessions per user, and any defined goals.

> - **Ads**: Provides an overview of your ad campaigns (i.e., Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, CTR (Click-Through Rate), CPC (Cost Per Click), CAC (Customer Acquisition Cost), and ROAS (Return on Ad Spend), plus flexible settings for acquisition and revenue tracking.

Author: ```suggestion
- **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, CTR (Click-Through Rate), CPC (Cost Per Click), CAC (Customer Acquisition Cost), and ROAS (Return on Ad Spend), plus flexible settings for acquisition and revenue tracking.
```

> - **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, CTR (Click-Through Rate), CPC (Cost Per Click), CAC (Customer Acquisition Cost), and ROAS (Return on Ad Spend), plus flexible settings for acquisition and revenue tracking.

Author: ```suggestion
- **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.
```

> - **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.

Author: Is there a way to mark just this section as Beta?

I saw this was used elsewhere:

```
{{partial:admonition type="beta" heading=""}}
This feature is in beta and is in active development. Features and functionality may change during the course of the beta period.
{{/partial:admonition}}
```

but it looks like it's too big -- would apply to an entire section, not just one bullet.

> - **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.

Author: Maybe just:

```suggestion
- **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking. (Closed Beta)
```

Reviewer: @Avery2 Are we looking for participants to the beta? If not, it's best to leave omit that until it's at least in open beta.

Author: Sorry @markzegarelli it should say "Open Beta"

> - **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.

Author: ```suggestion
- **Ad Performance**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.
```

> - **Ads**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.

Reviewer: ```suggestion
- **Ads** (Open Beta): Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.
```

> - **Ad Performance**: Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.

Reviewer: ```suggestion
- **Ad Performance** (Open Beta): Provides an overview of your ad campaigns (for example, Google Ads and Facebook Ads) with key metrics like Impressions, Clicks, Click-Through Rate (CTR), Cost Per Click (CPC), Customer Acquisition Cost (CAC), and Return on Ad Spend (ROAS), plus flexible settings for acquisition and revenue tracking.
```
